01week/datatypes.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+// logs the current date and time
+let displayDate = function () {
+  let date = new Date();
+  console.log(date);
+}
+displayDate();
+// takes string and converts it to integer
+let strConvert = function() {
+  let testString = '10';
+  console.log('the variable is set as ' + (typeof(testString)) + ' ' +testString);
+  let newInt = Number(testString);
+  console.log('now it is the ' +(typeof(newInt)) + ' ' +newInt);
+}
+strConvert();
+// takes integer and converts to a string
+let intConvert = function() {
+  let testInt = 9;
+  console.log('the variable is set as the ' +(typeof(testInt)) + ' ' +testInt);
+  let newStr =testInt.toString();
+  console.log('now it is the ' +(typeof(newStr)) + ' ' +newStr);
+}
+intConvert();
+// tests the datatype of each variable
+let dataType = function() {
+  let testBoolean = true;
+  console.log('true is of the datatype: ' +typeof(testBoolean));
+  let testNull = null;
+    if (testNull === null){
+    console.log('testNull is of the datatype: ' +testNull);
+    }
+  let testUnd;
+  console.log('no assigned value produces the datatype: ' +typeof(testUnd));
+  let testNum = 5;
+  console.log('5 is of the datatype: ' +typeof(testNum));
+  let testNaN = parseInt("LOL");
+  console.log('parseInt("LOL") produces the datatype: ' +(testNaN));
```

Comment:
This did not really test if a variable is an NaN. see the isNaN function: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN